frontend/src/models/dashboardsModel.tsx

```diff
@@ -69,6 +70,33 @@ export const dashboardsModel = kea<dashboardsModelType>({
                 }
                 return result
             },
+            copyDashboard: async ({ id }) => {
+                const originalDashboard = (await api.get(`api/dashboard/${id}`)) as DashboardType
+
+                const copiedItems = originalDashboard.items
+                    .filter((item) => !item.deleted)
+                    .map((item) => {
+                        const {
+                            result: _result, // eslint-disable-line
```

Comment:
I feel like this duplication functionality should be handled in the backend instead? For example here, you're creating a copy of all the dashboard items but are their `dashboard` relation keys pointing to the newly duplicated dashboard?